Worker.Extensions.Sql/src/packages.lock.json

```diff
@@ -199,8 +201,11 @@
       },
       "System.Diagnostics.DiagnosticSource": {
         "type": "Transitive",
-        "resolved": "5.0.0",
-        "contentHash": "tCQTzPsGZh/A9LhhA6zrqCRV4hOHsK90/G7q3Khxmn6tnB1PuNU0cRaKANP2AWcF9bn0zsuOoZOSrHuJk6oNBA=="
+        "resolved": "6.0.1",
```

Comment:
We can't do this, right?

Reply:
The minor bump of Azure.Core from 1.31 to 1.35 bumped this major version. I'll reach out to the Func host team to see what we can do here.

Reply:
Confirmed with Fabio that this bump is okay.